openssh/files/ssh_config

```diff
@@ -1,29 +1,71 @@
 {%- from "openssh/map.jinja" import client with context %}
```

Comment:
Use `-%}` here to strip empty lines at the beginning of file.

Reply:
Done in 7de3458.